tests/page-objects/default-mobile/search/search.wdio.page.js

```diff
@@ -0,0 +1,14 @@
+const searchPageDefault = require('@page-objects/default/search/search.wdio.page');
+
+const openSearchBox = () => $('.mm-search-bar-container .search-bar-left-icon .fa-search');
+
+const performSearch = async (term) => {
+  await (await openSearchBox()).waitForClickable();
+  await (await openSearchBox()).click();
+  await searchPageDefault.performSearch(term);
+};
+
+module.exports = {
+  searchPageDefault,
```

Comment:
I wonder if this separation is necessary. Why not just do:
```suggestion
...searchPageDefault
```

Reply:
Yes, I was following the style from this other [page object](https://github.com/medic/cht-core/blob/b98c1ff90370c9257639becb9ee3668d6415957b/tests/page-objects/default-mobile/reports/reports.wdio.page.js#L74). I remember that QA was thinking on restructuring the mobile e2e, I prefer to follow the style and wait to see how they will restructure page objects, before introducing a new way of doing this.
Since there aren't that many, it can be done in a different PR. 

cc: @tatilepizs @ngaruko